tools/sigh.ts

```diff
@@ -139,6 +139,8 @@ const nodeFlags = [
 ];
 
 // The 'cron' env type indicates the daily build in Travis.
+// TODO(bgogul): Review Travis-specific health stats upload and remove or
+// rework for Parker.
 const isTravisDaily = (process.env.TRAVIS_EVENT_TYPE === 'cron');
```

Comment:
@bgogul FYI on this, I hadn't realized we have this logic. There's some stuff that looked for Travis-specific runs and would upload project health stats. We could file a P2 to look at this later.

Reply:
I did see them. I thought it was similar to the gcb dashboard, which has average run times and other stats:  https://console.cloud.google.com/cloud-build/dashboard?project=arcs-265404

In any case, I can take a look at it. Should we file an issue?

Reply:
Filed https://github.com/PolymerLabs/arcs/issues/4757